src/object_functions.c

```diff
@@ -1043,7 +1029,24 @@ void obj_init_posarrow(Object *obj, LevelObjectEntry_PosArrow *entry) {
     obj->unk6 |= 0x4000;
 }
 
-GLOBAL_ASM("asm/non_matchings/unknown_032760/obj_loop_posarrow.s")
+void obj_loop_posarrow(Object *obj, s32 speed) {
+    Object_64_PosArrow *someObj64;
+    Object **someObjList;
+    Object *someObj;
+    s32 numberOfObjects;
+
+    obj->unk6 |= 0x4000;
+    someObjList = func_8001BAAC(&numberOfObjects);
+    if (obj->unk78 < numberOfObjects) {
+        someObj = someObjList[obj->unk78];
+        someObj64 = someObj->unk64;
+        if (someObj64->unk0 == -1) {
+            obj->unk6 &= 0xBFFF;
```

Comment:
```suggestion
            obj->unk6 &= ~0x4000;
```
IDK what this represents, but I'd guess this is the intent.